cypress/integration/SurveyDetail/index.spec.ts

```diff
@@ -42,9 +42,13 @@ describe('Survey detail page', () => {
 
       cy.findByRole('button', { name: /Start Survey/ }).click();
 
+      cy.findByText('Food â€“ Variety, Taste and Presentation', {
```

Comment:
I forgot to ask earlier, but is there an encoding issue? Shouldn't it be something like `Food & Variety, ...`? 🤔

Reply:
It's from API response. I tried to update `Content-Type` and `Accept-charset` in the request headers, but they still display invalid encoding. I think we should handle this encoding from the API side 🤔

![image](https://user-images.githubusercontent.com/14077479/179712771-e3ab60e0-b2a3-45bc-8b5d-11071ef6aea4.png)

Reply:
Yup, that seems to be a bug on the API then :D 
thanks for your explanation :)